lua/damagelogs/config/config.lua

```diff
@@ -49,36 +50,36 @@ REQUIRES ULX ! If you are using ServerGuard, set this to 0 (it will use ServerGu
 ]]
 --
 Damagelog.ULX_AutoslayMode = 1
--- Force autoslain players to be innocents (ULX only)
+-- Force autoslain players to be innocents (ULX/SAM only)
 -- Do not enable this if another addon interferes with roles (Pointshop roles for example)
 Damagelog.ULX_Autoslay_ForceRole = true
--- Default autoslay reasons (ULX and ServerGuard)
-Damagelog.Autoslay_DefaultReason = "broke server rules"
-Damagelog.Autoslay_DefaultReason1 = "random kill"
-Damagelog.Autoslay_DefaultReason2 = "multiple random kills"
-Damagelog.Autoslay_DefaultReason3 = "random damage"
-Damagelog.Autoslay_DefaultReason4 = "multiple random damage"
-Damagelog.Autoslay_DefaultReason5 = "teamkill"
-Damagelog.Autoslay_DefaultReason6 = "needless report"
-Damagelog.Autoslay_DefaultReason7 = "unfitting answer"
-Damagelog.Autoslay_DefaultReason8 = "unfitting language"
-Damagelog.Autoslay_DefaultReason9 = "lying"
-Damagelog.Autoslay_DefaultReason10 = "propkill"
-Damagelog.Autoslay_DefaultReason11 = "teaming"
-Damagelog.Autoslay_DefaultReason12 = "random kos"
+-- Default autoslay reasons (ULX, SAM, and ServerGuard)
+Damagelog.Autoslay_DefaultReason1 = "Random Damage"
+Damagelog.Autoslay_DefaultReason2 = "RDM"
+Damagelog.Autoslay_DefaultReason3 = "2x RDM"
+Damagelog.Autoslay_DefaultReason4 = "Attempted Mass"
+Damagelog.Autoslay_DefaultReason5 = "Mass RDM"
+Damagelog.Autoslay_DefaultReason6 = "Super Mass"
+Damagelog.Autoslay_DefaultReason7 = "Ghosting"
+Damagelog.Autoslay_DefaultReason8 = "Hacking"
+Damagelog.Autoslay_DefaultReason9 = "Prop kill"
+Damagelog.Autoslay_DefaultReason10 = "Consistent RDM"
+Damagelog.Autoslay_DefaultReason11 = "Yolo"
+Damagelog.Autoslay_DefaultReason12 = "Yolo"
```

Comment:
I think most of these are better than the original default reasons.
Could we replace the two "Yolo" reasons with something else?

`Damagelog.Autoslay_DefaultReason` will need re-adding, as this is used in some parts of the code

Reply:
done